src/stylesheets/components/_navigation-letter.scss

```diff
@@ -11,18 +11,14 @@
     margin-left: 0;
     float: left;
     padding: 0;
-    line-height: ($base-spacing-unit-large - 4);
     text-align: center;
     text-transform: uppercase;
     &:last-child {
       margin-right: 0;
     }
-    & > a {
-      display: block;
-    }
     @include media($desktop) {
-    	width: 7.69230769231%;
-      line-height: ($base-spacing-unit-large - 10);
+      //width: 7.69230769231%;
+      width: 3.84615384615%;
```

Comment:
Makes the a-z spread across 1 line or 2 lines - currently set to spread across 1 line but it may well go back to 2